core/whistle_apps-1.0.0/src/whapps_util.erl

```diff
@@ -284,6 +285,24 @@ is_account_db(Db) -> couch_util:db_classification(Db) =:= 'account'.
 
 %%--------------------------------------------------------------------
 %% @public
+%% @doc Ids are one->one with accounts.
+%% @end
+%%--------------------------------------------------------------------
+-spec get_account_by_id(ne_binary()) ->
+                                  {'ok', ne_binary()} |
+                                  {'error', 'not_found'}.
+get_account_by_id(Id) ->
+    case couch_mgr:open_doc(?WH_ACCOUNTS_DB, Id) of
```

Comment:
Take a look at the "do_get_accounts_by" function that both get_account_by_realm and get_account_by_name call. They both return the account dbs for the parameter.